src/utils/helpers.ts

```diff
@@ -45,3 +54,71 @@ export const isValidDate = (d: string) => {
   }
   return false;
 };
+
+/**
+ * @param experience - an attorney's required experience for a listed case.
+ * @returns appropriate string description according to the experience level.
+ */
+export const parseExperience = (experience: string) => {
+  if (experience === 'LOW') {
+    return `${
+      experience.charAt(0).toUpperCase() + experience.toLowerCase().substring(1)
+    } (No prior case)`;
+  }
+  if (experience === 'MEDIUM') {
+    return `${
+      experience.charAt(0).toUpperCase() + experience.toLowerCase().substring(1)
+    } (One prior case)`;
+  }
+  return `${
+    experience.charAt(0).toUpperCase() + experience.toLowerCase().substring(1)
+  } (Multiple prior cases)`;
+};
+
+/**
+ * If the description is used for the case listing cards, CARD should be passed in as
+ * TRUE, FALSE otherwise.
+ * @param attorney - a boolean indicating whether a case needs an attorney.
+ * @param interpreter - a boolean indicating whether a case needs an interpreter.
+ * @param card a boolean indicating whether the description will be used for a case listing card.
+ * @returns appropriate description for roles needed field based on the attorney & interpreter parameters.
+ */
+export const parseRolesNeeded = (
+  attorney: boolean | undefined,
+  interpreter: boolean | undefined,
```

Comment:
instead of `boolean | undefined`, it may be cleaner to write something like `attorney?: boolean`